test/functional/tests/incremental_load/test_inactive_cores.py

```diff
@@ -16,152 +18,192 @@
+
+        inactive_cores_occupancy_stats = 0
+        inactive_cores_clean_stats = 0
+        inactive_cores_dirty_stats = 0
+
+        inactive_cores = get_inactive_cores(cache_id=cache.cache_id)
+        for core in inactive_cores:
+            core_stats = core.get_statistics()
+            inactive_cores_occupancy_stats += core_stats.usage_stats.occupancy
+            inactive_cores_clean_stats += core_stats.usage_stats.clean
+            inactive_cores_dirty_stats += core_stats.usage_stats.dirty
+
+        cache_stats = cache.get_statistics()
+        cache_usage_stats = cache_stats.usage_stats
+
+        total_cores_occupancy_stats = active_cores_occupancy_stats + inactive_cores_occupancy_stats
+        total_cores_dirty_stats = active_cores_dirty_stats + inactive_cores_dirty_stats
+        total_cores_clean_stats = active_cores_clean_stats + inactive_cores_clean_stats
+
+        if cache_usage_stats.occupancy != total_cores_occupancy_stats:
+            TestRun.LOGGER.error("Wrong number of occupancy blocks in cache usage stats")
+        if cache_usage_stats.dirty != total_cores_dirty_stats:
+            TestRun.LOGGER.error("Wrong number of dirty blocks in cache usage stats")
+        if cache_usage_stats.clean != total_cores_clean_stats:
+            TestRun.LOGGER.error("Wrong number of clean blocks in cache usage stats")
+
+        if cache_usage_stats.inactive_occupancy != inactive_cores_occupancy_stats:
+            TestRun.LOGGER.error("Wrong number of occupancy blocks in inactive cache usage stats")
+        if cache_usage_stats.inactive_dirty != inactive_cores_dirty_stats:
+            TestRun.LOGGER.error("Wrong number of dirty blocks in cache inactive usage stats")
+        if cache_usage_stats.inactive_clean != inactive_cores_clean_stats:
+            TestRun.LOGGER.error("Wrong number of clean blocks in cache inactive usage stats")
+
+        cache_usage_stats_percentage = cache.get_statistics(percentage_val=True).usage_stats
+
+        # Calculate expected percentage value of inactive core stats
+        inactive_occupancy_perc = round(
+            100 * (cache_usage_stats.inactive_occupancy / cache_stats.config_stats.cache_size), 1
+        )
+        inactive_dirty_perc = round(
+            100 * (cache_usage_stats.inactive_dirty / cache_stats.usage_stats.occupancy), 1
+        )
+        inactive_clean_perc = round(
+            100 * (cache_usage_stats.inactive_clean / cache_stats.usage_stats.occupancy), 1
+        )
+
+        if inactive_occupancy_perc != cache_usage_stats_percentage.inactive_occupancy:
+            TestRun.LOGGER.error("Wrong occupancy percentage in usage stats")
+        if inactive_dirty_perc != cache_usage_stats_percentage.inactive_dirty:
+            TestRun.LOGGER.error("Wrong dirty blocks percentage in usage stats")
+        if inactive_clean_perc != cache_usage_stats_percentage.inactive_clean:
+            TestRun.LOGGER.error("Wrong clean blocks percentage in usage stats")
```

Comment:
if TestRun.LOGGER.error occurs, please print matching statistics in message log. (it applies to the whole test)